packages/elements/src/components/ui/button-links/PrimaryButtonLink.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import cx from "classnames";
+import styles from "../buttons/Button.module.css";
+import { AnchorElementProps } from "@stenajs-webui/core";
+import { CommonButtonProps } from "../buttons/common/ButtonCommon";
+import { getButtonLabel } from "../buttons/common/ButtonLabelFactory";
+import { ButtonContent } from "../buttons/common/ButtonContent";
+
+export interface PrimaryButtonLinkProps
+  extends CommonButtonProps,
+    AnchorElementProps {}
+
+export const PrimaryButtonLink: React.FC<PrimaryButtonLinkProps> = ({
+  label,
+  className,
+  size = "normal",
+  loadingLabel,
+  loading = false,
+  success = false,
+  successLabel,
+  leftIcon,
+  left,
+  rightIcon,
+  right,
+  variant = "normal",
+  innerRef,
+  ...anchorProps
+}) => {
+  const buttonLabel = getButtonLabel(
+    label,
+    success,
+    successLabel,
+    loading,
+    loadingLabel
+  );
+
+  const hasLabel = Boolean(
+    (label && !success && !loading) ||
+      (success && successLabel) ||
+      (loading && loadingLabel)
+  );
+
+  return (
+    <a ref={innerRef} className={styles.buttonLink} {...anchorProps}>
+      <div
```

Comment:
This works but I'm just curious why we don't use the cx classes directly on the `a`? :)

Reply:
Updated!